core/objects.go

```diff
@@ -553,6 +556,95 @@ type OCSPSigningRequest struct {
 	RevokedAt time.Time
 }
 
+type SignedCertificateTimestamp struct {
+	ID int `db:"id",json:"-"`
```

Comment:
I know it's a pain in the ass, but please make a db model for this separate from the internal API representation. We're getting burned because these things tend to grow right now.